man/udevadm.xml

```diff
@@ -319,6 +319,19 @@
             the same command to finish.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>--ping</option></term>
+          <listitem>
+            <para>Before triggering events, check that <command>systemd-udevd</command> daemon is already started.</para>
```

Comment:
I think it's better not to add two options for this. Just add `--timeout=` or `--daemon-timeout=`, and if it is specified, wait the specified time. The fact that this is implemented as a ping message is just an internal naming detail. The description of the option needs to say that this waits for the daemon to start.